<!--
 * This file is part of Blind Camaleon.
 *
 * Blind Camaleon is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Blind Camaleon is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with Blind Camaleon.  If not, see <http://www.gnu.org/licenses/lgpl-3.0.txt>.
-->

<ui version="4.0" >
 <class>PIDMainWindow</class>
 <widget class="QMainWindow" name="PIDMainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Blind Camaleon v1.0a</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QMdiArea" name="mdiArea" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="menuOpenImage" />
    <addaction name="menuSaveCurrent" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>&amp;Image</string>
    </property>
    <addaction name="menuGenerateGrayScale" />
    <addaction name="menuGenerateHistogram" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="separator" />
    <addaction name="menuOpenHelp" />
    <addaction name="menuOpenAbout" />
   </widget>
   <widget class="QMenu" name="menuFilters" >
    <property name="title" >
     <string>Filters</string>
    </property>
    <widget class="QMenu" name="menuButterworth" >
     <property name="title" >
      <string>Butterworth</string>
     </property>
     <addaction name="menuButterworthAlta" />
     <addaction name="menuButterworthBaixa" />
    </widget>
    <addaction name="menuFiltersAltaIdeal" />
    <addaction name="menuFiltersBaixaIdeal" />
    <addaction name="menuFiltersIdeal" />
    <addaction name="menuButterworth" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuTools" />
   <addaction name="menuFilters" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <action name="menuOpenImage" >
   <property name="text" >
    <string>&amp;Open Image</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="menuGenerateGrayScale" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Generate &amp;Gray Scale</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="menuSaveCurrent" >
   <property name="text" >
    <string>&amp;Save Current As...</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="menuOpenHelp" >
   <property name="text" >
    <string>&amp;Help</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
  </action>
  <action name="menuOpenAbout" >
   <property name="text" >
    <string>&amp;About Blind Camaleon</string>
   </property>
  </action>
  <action name="menuGenerateHistogram" >
   <property name="text" >
    <string>Generate &amp;Histogram</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="menuGenerateSpectrum" >
   <property name="text" >
    <string>Generate &amp;Fourier Spectrum</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="menuFiltersAltaIdeal" >
   <property name="text" >
    <string>Passa alta</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P, A</string>
   </property>
  </action>
  <action name="menuFiltersBaixaIdeal" >
   <property name="text" >
    <string>Passa baixa</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P, B</string>
   </property>
  </action>
  <action name="menuFiltersIdeal" >
   <property name="text" >
    <string>Ideal</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="menuButterworthAlta" >
   <property name="text" >
    <string>Passa alta</string>
   </property>
  </action>
  <action name="menuButterworthBaixa" >
   <property name="text" >
    <string>Passa baixa</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMdiArea</class>
   <extends>QFrame</extends>
   <header location="global" >QMdiArea</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
